RetailCoder.VBE/Root/RubberduckModule.cs

```diff
@@ -425,6 +429,10 @@ private IEnumerable<ICommandMenuItem> GetRubberduckCommandBarItems()
 
         private IEnumerable<IMenuItem> GetRubberduckMenuItems()
         {
+            //This bind needs to occur before the the new array is built by the function calls
```

Comment:
That's because the factory interface naming convention is kicking in and binding to an automatic factory implementation; if you want to override that binding you can use `Rebind` - please keep `GetRubberduckMenuItems` dedicated to returning the RD menu items.